cluster/images/hyperkube/Makefile

```diff
@@ -53,57 +53,86 @@ build:
 ifndef VERSION
     $(error VERSION is undefined)
 endif
-	cp -r ./* ${TEMP_DIR}
-	mkdir -p ${TEMP_DIR}/cni-bin ${TEMP_DIR}/addons ${TEMP_DIR}/addons/singlenode ${TEMP_DIR}/addons/multinode
+
```

Comment:
Ok, I left 2 empty lines between sections to give to code a bit structure. Ok now?